### PR TITLE
Remove case token listing from contact email task

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Remove case token listing from contact email task

Before
----------------------------------------
Contact email task calls the legacy function

After
----------------------------------------
Contact email task calls the token processor (exposes domain tokens)

Technical Details
----------------------------------------
The only reason not to call the token processor to get these is if you are trying to filter the tokens by 'case_type_id' - which should now only apply to the case email task

Comments
----------------------------------------
@demeritcowboy this was a side-track on my path to figure out the rows issue

### CODE REVIEW

**Comment by totten** [comment]
For `r-run`, I did a few scenarios:

| Use-Case | Before Patch | After Patch |
| -- | -- | -- |
| (1) "View Contact => Send Email" | No case tokens | No case tokens |
| (2) "Find Contacts => Select All => Email - send now" | No case tokens | No case tokens |
| (3) "Manage Case => Email (icon)" |  Case tokens present | Case tokens present |

It appears to me that they behave the same. I pulled up a debugger and confirmed that (3) in fact goes through `CRM/Case/Form/Task/Email.php` (which didn't exist at the time of #13473). So in effect that snippet has been successfully relocated from `CRM/Contact/Form/Task/Email.php` to  `CRM/Case/Form/Task/Email.php`

It's possible that the change means the contact-flavored screen will get a fuller list of tokens by way of `TokenProcessor`, which sounds good.

tldr: Change looks fine to me.
